### PR TITLE
Created all the Functions

### PR DESCRIPTION
**Pull request by itsnestee**
Added ten functions.

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -60,10 +59,21 @@ void variableDeclarations()
...
-    
-
-    
-    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+    
+    int siblings = 4;
+    bool isTwoAnEvenNumber = 1;
+    double myBodyTemperature = 36.5;
+    float pi = 0.1f;
+    char grade = 'C';
```
Comment (matkatmusic) [inline main.cpp:67]:
```
2) for each primitive type, write out 3 variable declarations
```
- [ ] revise

File: main.cpp [hunk 2]
```diff
@@ -60,10 +59,21 @@ void variableDeclarations()
...
+
+    
+    ignoreUnused(number);
+    ignoreUnused(siblings);
+    ignoreUnused(isTwoAnEvenNumber);
+    ignoreUnused(myBodyTemperature);
+    ignoreUnused(pi);
+    ignoreUnused(grade);
```
Comment (matkatmusic) [inline main.cpp:75]:
```
    b) at the end of the function, call ignoreUnused ONCE and pass ALL of your variables to it.
```
- [ ] revise

File: main.cpp [hunk 3]
```diff
@@ -136,28 +193,40 @@ int main()
...
+    ignoreUnused(drumsPlayed);
+    ignoreUnused(rainingOutside);
+    ignoreUnused(buttonPressed);
+    ignoreUnused(purchasedProduct);
+    ignoreUnused(finishedBreak);
+    ignoreUnused(doneWalking);
+    ignoreUnused(repairedPhone);
+    ignoreUnused(cleanKitchen);
+    ignoreUnused(landedAbroad);
+    ignoreUnused(masteringCPlusPlus);
```
Comment (matkatmusic) [inline main.cpp:229]:
```
        c) pass the local variables to ignoreUnused() as you did in 2b)
```
as a reminder of what 2b) said:
```
    call ignoreUnused ONCE and pass ALL of your variables to it.
```
- [ ] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]
The indentation for all of your functions is incorrect.
please read the Readme.MD, look for the section explaining the Course Coding Standard and conform to the Course Coding Standard.
- [x] revise

File: main.cpp (hunk 1 above)
Comment (itsnestee) [reply main.cpp:67]: Ok, example. 
as I have int siblings = 4; I should do something like;
siblings = 5;
siblings = 7;
siblings = 15;  is this what you mean by creating 3 variable declarations?

File: main.cpp [hunk 4]
```diff
@@ -60,10 +59,37 @@ void variableDeclarations()
...
+
+    float pi = 3.14f;
+          pi = 6.28f;
+          pi = 9.42f;
+          pi = 12.56f;
+
+    char grade = 'C';
+         grade = 'A';
+         grade = 'B';
+         grade = 'D';
```
Comment (matkatmusic) [inline main.cpp:87]: This is not declaring 3 variables of a particular type. and assigning them an initial value.
This is declaring 1 variable of a particular type and changing its value 4 times.

The purpose of this assignment is to teach you how to declare variables and initialize them with values.
Also, do not initialize boolean variables with `0` or `1`.   Be explicit when you initialize booleans and use `true` or `false`.
- [ ] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]
This is a list of all the things I need you to fix in this project.
- Do NOT initialize boolean variables with `0` or `1`.  use `true` or `false`
- fix your indentation.  Set up Replit to use 4 spaces when you press the 'tab' key.
- do not put spaces around `(` or `)` 
- do not add extra blank lines inside your functions unless they improve readability.
- write `return {};` not `return{};`.  it's easier to read.
- [ ] revise

File: main.cpp (hunk 4 above)
Comment (itsnestee) [reply main.cpp:87]: Can you kindly show me an example?

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**